api/src/index.js

```diff
@@ -11,21 +11,87 @@ app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
-apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
+//handle no data found
+const handleNoData = (req, res, next) => {
+  if (req.data && req.data.length === 0) {
+    return res.status(404).json({ message: "No meals found" });
+  }
+  next(); 
+};
+
+//handle errors
+const errorHandler = (err, req, res, next) => {
+  console.error("Error:", err.message);
+  res.status(err.status || 500).json({ error: err.message });
+};
+
+// Route to get all future meals
+app.get('/future-meals', async (req, res, next) => {
+  try {
+    const now = new Date();
+    const futureMeals = await knex.raw('Meal').where('when', '>', now).select('*');
```

Comment:
this should be in following order:
```
knex('Meal').select('*').('when', '>', now)
```

Reply:
I will do the changes, thanks for your sujjections.